src/containers/AppBody.jsx

```diff
@@ -96,13 +97,10 @@ export class AppBody extends Component {
       .then((res) => {
         // Normalize the app state received from S3
         const appState = setDefaults(removeDeprecatedFiltersFromState(res.data));
```

Comment:
This probably should be refactored away, since you're using transformation functions with axios.